website/docs/reference/runtimes/index.mdx

```diff
@@ -15,14 +17,41 @@ The status represents the maturity of the runtime implementation:
 Submit your request and vote for your preferred ones [here](https://github.com/metatypedev/metatype/discussions/305).
 :::
 
-| Runtime                         | Description                                                | Runtime version | Status | Typegate version |
+|             Runtime             |                        Description                         | Runtime version | Status | Typegate version |
 | ------------------------------- | ---------------------------------------------------------- | --------------- | ------ | ---------------- |
 | [Prisma](./runtimes/prisma)     | Query SQL and no-SQL databases with Prisma                 | 4.17.0          | beta   | >0.1.0           |
 | [Deno](./runtimes/deno)         | Execute Typescript functions with NPM or deno dependencies | 1.35.0          | beta   | >0.1.0           |
 | [HTTP](./runtimes/http)         | Make HTTP requests to (REST) HTTP APIs                     |                 | beta   | >0.1.0           |
 | [GraphQL](./runtimes/graphql)   | Make GraphQL requests                                      |                 | beta   | >0.1.0           |
 | [Random](./runtimes/random)     | Generate random data based on your schema                  |                 | alpha  | >0.1.0           |
 | [S3](./runtimes/s3)             | Manipulate S3-compatible storage                           |                 | alpha  | >0.1.0           |
-| [Temporal](./runtimes/temporal) | Query and trigger workflow orchestrated by Temporal        |                 | alpha  | >0.1.0           |
-| [WasmEdge](./runtimes/wasmedge) | Execute WebAssembly functions                              |                 | alpha  | >0.1.0           |
-| [Python](./runtimes/python)     | Execute Python functions                                   |                 | alpha  | >0.1.0           |
+| [Temporal](./runtimes/temporal) | Query and trigger workflow orchestrated by Temporal        |                 | alpha  | >=0.0.3-dev.1    |
+| [Wasm](./runtimes/wasm)         | Execute WebAssembly functions                              |                 | alpha  | >=0.0.3-dev.6    |
+| [Python](./runtimes/python)     | Execute Python functions                                   |                 | alpha  | >=0.1.14         |
+
+## Tiers
+In total, metatype aims to support 3 types of runtimes, each type corresponding to a tier.
+
+### Tier 1 - Embedded runtimes
+User defined runtimes, you can write your own runtime logic and package it along with your typegraph.\
+This type of runtime refers to any runtime that uses the materializer development kit, which can be generated from the `meta gen mdk` command.
+
+
+### Tier 2 - Embedded language runtimes
+This type of runtime covers any runtime that uses a sandboxed programming language runtime that you can define custom logic on.
+
+This includes:
+- Deno
+- Python
+- Wasm
+
+### Tier 3 - Specialized runtimes
```

Comment:
On second blush, the tier terminology here seems out of place. Tiers usually indicate status of support with expectation being the higher the tier, the better of support. The usage here seems to be more about classes/types of runtimes. @zifeo thoughts?